spec/features/trainees/edit_course_details_spec.rb

```diff
@@ -75,19 +71,22 @@ def when_i_visit_the_course_details_page
   end
 
   def and_i_enter_valid_parameters
-    course_details_page.subject.select(template.subject)
-    course_details_page.set_date_fields("course_start_date", template.course_start_date.strftime("%d/%m/%Y"))
-    course_details_page.set_date_fields("course_end_date", template.course_end_date.strftime("%d/%m/%Y"))
-    age_range = Dttp::CodeSets::AgeRanges::MAPPING[template.age_range]
+    course_details_page.subject.select(trainee.subject)
+    course_details_page.set_date_fields("course_start_date", trainee.course_start_date.strftime("%d/%m/%Y"))
+    course_details_page.set_date_fields("course_end_date", trainee.course_end_date.strftime("%d/%m/%Y"))
+
+    age_range = Dttp::CodeSets::AgeRanges::MAPPING[trainee.course_age_range]
 
     if age_range[:option] == :main
-      course_details_page.public_send("main_age_range_#{template.age_range.parameterize(separator: '_')}").choose
+      course_details_page.public_send("main_age_range_#{trainee.course_age_range.join('_to_')}").choose
     else
       course_details_page.main_age_range_other.choose
-      course_details_page.additional_age_range.select(template.age_range)
+      course_details_page.additional_age_range.select(trainee.course_age_range.join(" to "))
     end
   end
 
+  def and_i_enter_invalid_parameters; end
```

Comment:
Empty method :(

Reply:
Good spot - I was intending on fleshing it out, but didnt need to in the end.